shotover-proxy/benches/benches/chain.rs

```diff
@@ -272,6 +275,7 @@ fn criterion_benchmark(c: &mut Criterion) {
         });
     }
 
+    #[cfg(feature = "alpha-transforms")]
```

Comment:
These are needed now because ProtectConfig is only available when alpha-transforms is enabled